certification/internal/shell/base_on_ubi_test.go

```diff
@@ -1,41 +1,65 @@
 package shell
```

Comment:
Is it possible to move the tests to a separate directory?

Reply:
@samira-barouti Technically, yes! For example, an integration test might live elsewhere and don't really need to live next to packages.

With that said, storing test code for a package alongside the package is idiomatic for most unit test cases from what I've seen. Also the below snippet from the golang testing docs (though we use Ginkgo which integrates well with the go test tooling):

> To write a new test suite, create a file whose name ends _test.go that contains the TestXxx functions as described here. Put the file in the same package as the one being tested. The file will be excluded from regular package builds but will be included when the "go test" command is run. For more detail, run "go help test" and "go help testflag".
reference: https://golang.org/pkg/testing/

Storing the test also gives @bcrochet the ability to leverage any local-only variables/functions etc. that he might need or want to test. I think the only one that is in scope here is the podman engine.

As an aside (and not contextually related to this PR), you can also create a test file in the same package using a new package name `<packagename>_test` which gives you the ability to do black-box testing, and only focus on exported functionality.

Reply:
Understood. I was just thinking as we add more checks, this directory will become cumbersome . I am good with it at the moment, however we need to start splitting this package into subpackages (probably leveraging container vs bundle sub packages).

Reply:
The way that I'm doing the mocking would break if we moved the tests to a different directory/package. As @komish pointed out, this is the idiomatic way.